### DISCUSSION TITLE
Is there an API to switch the default coordinate system

### DISCUSSION THREAD

**Discussion by evilzhou** [pyvista/pyvista#3949] [Q&A]
Is it possible to switch between left-hand and right-hand coordinate system, otherwise the model is mirrored

**Comment by banesullivan** [comment]
AFAIK, this isn't changeable in VTK and you have to flip/transform your data

There's a VTK post on this: https://discourse.vtk.org/t/change-coordinate-system-from-right-to-left-handed-change-direction-of-z-axis/4635

See also: https://docs.pyvista.org/api/core/_autosummary/pyvista.DataSetFilters.transform.html#transform
